samples/mysql-schema/pom.xml

```diff
@@ -30,10 +30,6 @@
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-container-image-jib</artifactId>
     </dependency>
-    <dependency>
```

Comment:
How can it compile when it is still used ?

https://github.com/quarkiverse/quarkus-operator-sdk/blob/1d7c65a9c70a251ed84f66e869ccbc2d3a0f46ca/samples/mysql-schema/src/main/java/io/quarkiverse/operatorsdk/samples/mysqlschema/MySQLSchemaReconciler.java#L11

Reply:
Hmm, actually, I made the wrong change, I was planning on leaving the samples alone. 😅
That said the dependency is probably coming transitively…

Reply:
yes it i brought by io.quarkus:quarkus-core-deployment.
+1 for letting it in the sample :)

Reply:
By the way, I'm having issues with the MySQL test locally: it times out here. I'm wondering if there's an issue with Docker and testcontainers on M1 macs…

Reply:
> By the way, I'm having issues with the MySQL test locally: it times out here. I'm wondering if there's an issue with Docker and testcontainers on M1 macs…

@metacosm On some environnement (especially when the docker daemon is not on the host but in a wrapper VM like with Docker Desktop for win/mac), exposed containers are not accessible on localhost and you need to set an env var `TESTCONTAINERS_HOST_OVERRIDE=host.docker.internal` with the correct hostname.